dango/dex/src/cron.rs

```diff
@@ -419,6 +419,11 @@ fn clear_orders_of_pair(
     let mut inflows = DecCoins::new();
     let mut outflows = DecCoins::new();
 
+    #[cfg(feature = "metrics")]
+    let mut metric_volume: HashMap<(&Denom, &Denom, &Denom), grug::Uint128> = HashMap::new();
+    #[cfg(feature = "metrics")]
+    metrics::counter!("dex.total_trades").increment(filling_outcomes.len() as u64);
```

Comment:
We should find an agreed namespace for contract related metrics like `dango.contract.dex.<...>`